build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -334,3 +334,26 @@ multicluster:
 # The default is antrea-agent's Namespace.
   namespace: {{ .namespace | quote }}
 {{- end }}
+
+secondaryNetwork:
+{{- with .Values.secondaryNetwork }}
+  # Secondary network OVS configuration
+  ovs:
+    # Secondary network OVS configuration enable
```

Comment:
Probably change this one to be consistent with values.yaml: Enable OVS bridge configuration for secondary network

Reply:
Got it aligned.